pkg/framework/aries/defaultopt/defaultopt.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package defaultopt
+
+import (
+	"fmt"
+
+	"github.com/hyperledger/aries-framework-go/pkg/framework/aries"
+	"github.com/hyperledger/aries-framework-go/pkg/storage/leveldb"
+)
+
+// WithDBPath return new default store provider instantiate with db path
+func WithDBPath(dbPath string) aries.Option {
```

Comment:
I think it would be better to use RFC language. I have my doubts that they called it "DB"?

Reply:
I change it to storePath https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0050-wallets/README.md#composition